graylog2-plugin-interfaces/src/main/java/org/graylog2/plugin/buffers/MessageEvent.java

```diff
@@ -45,6 +46,7 @@ public MessageEvent newInstance()
 
     private RawMessage raw;
     private Message msg;
```

Comment:
I kept that because the OutputBuffer is only using the single message field and does not need a list of messages.